### DISCUSSION TITLE
Error with @supabase/auth-helpers-nextjs tsconfig.json file

### DISCUSSION THREAD

**Discussion by adamjdavies** [supabase/supabase#13347] [Questions]
Hi all, 

I'm getting this error in the tsconfig.json file of the @supabase/auth-helpers-nextjs node module:

<img width="593" alt="image" src="https://user-images.githubusercontent.com/123465379/229163664-f056af09-450d-4acb-a84a-163badd9dfb7.png">

Any ideas how to fix it? It only appeared after I was having a look through the files. 

Thanks

**Comment by joecf** [comment]
> Hi all,
> 
> I'm getting this error in the tsconfig.json file of the @supabase/auth-helpers-nextjs node module:
> 
> <img alt="image" width="593" src="https://user-images.githubusercontent.com/123465379/229163664-f056af09-450d-4acb-a84a-163badd9dfb7.png">
> 
> Any ideas how to fix it? It only appeared after I was having a look through the files.
> 
> Thanks

I had this problem, just closing VS code and reopening worked for me, was a bug.
